src/Native/Common/pal_config.h.in

```diff
@@ -36,6 +36,9 @@
 #cmakedefine01 HAVE_TCP_VAR_H
 #cmakedefine01 HAVE_RT_MSGHDR
 #cmakedefine01 HAVE_LINUX_RTNETLINK_H
+#cmakedefine01 HAVE_INOTIFY_INIT
```

Comment:
I'm not suggesting that we don't just check the compatible functions: I'm suggesting that we check for compatible functions *and* error out on Linux only if `inotify` cannot be found.